src/ipfs.io/docs/news/index.html

```diff
@@ -0,0 +1,51 @@
+---
+baseurl: ../..
+site: ipfs
+template: tmpl/layouts/ipfs.html
+title: News
+---
+{% set pgns = 'news' %}
+{% include "tmpl/partials/pagenav.html" %}
+
+<div class="col-lg-6 col-lg-offset-3 col-sm-8 col-sm-offset-2" style="font-family: Oxygen;">
+{% markdown %}
+
+## News
+
+IPFS news, articles, references on the media, presence in conferences and talks, organized by chronological order.
+
+### 2015
+- [Oct 21 - Stanford EE380 - Computer Systems Colloquium](http://www-cs.stanford.edu/events/computer-systems-colloquium-ipfs-and-permanent-web-juan-benet-protocol-labs)
+- [Oct 20 - IPFS Meetup hosted by Sourcegraph, San Francisco CA](http://attending.io/events/ipfs-san-francisco-october-2015)
+- [Oct 19 - Introduction to IPFS (InterPlanetary File System); a brief post about a protocol that changes everything](https://www.linkedin.com/pulse/introduction-ipfs-interplanetary-file-system-brief-post-john-lilic?trk=hp-feed-article-title-like)
+- [Oct 18 - Ian Preston - IPFS at Redecentralize Conf, London, UK](https://www.youtube.com/watch?v=TiCUIh7tNtU)
+- [Oct 12 - Epicenter Bitcoin Interviews Juan Benet - Decentralizing The Web With The Inter-Planetary File System (IPFS)](https://epicenterbitcoin.com/podcast/100/)
+- [Oct 12 - Decentralizing the Web with IPFS](http://blog.resellerclub.com/decentralizing-the-web-with-ipfs/)
+- [Oct 9 - FreeNAS Alpha ships with IPFS as a transport](http://www.freenas.org/whats-new/2015/10/announcing-freenas-10-alpha.html)
+- [Oct 6-8 - AndyetConf - Richland, WA](http://andyetconf.com)[video (soon)]()
+- [Oct 4 - TechCruch - Why The Internet Needs IPFS Before It’s Too Late](http://techcrunch.com/2015/10/04/why-the-internet-needs-ipfs-before-its-too-late)
+- [Sep 23 - IPFS — Inter-Planetary File system. A for-life, decentralised, privacy respecting web.](https://medium.com/@mvxlr/ipfs-inter-planetary-file-system-65466e4129c6)
+- [Sep 18 - Motherboard, The InterPlanetary File System Wants to Create a Permanent Web](http://motherboard.vice.com/read/the-interplanetary-file-system-wants-to-create-a-permanent-web)
```

Comment:
Here, and elsewhere, we mix All Caps with Initial caps. Can we standardize this? Happy to copy edit.